src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,50 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION = 0;
+    private static final int AMOUNT = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int buy = 0;
+        int supply = 0;
+        String [] lineInfo;
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            String readLine = bufferedReader.readLine();
+            while (readLine != null) {
+                lineInfo = readLine.split(",");
```

Comment:
All magic numbers, strings, and symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)
```suggestion
                lineInfo = readLine.split(SEPARATOR);
```